data/application.go

```diff
@@ -37,5 +37,24 @@ var (
 	UploadBinary = api.Application {
 		Name: "upload-binary",
 	}
-	ApplicationSamples = []api.Application{Minimal, PathfinderGit, BookServer, TackleTestapp, UploadBinary}
+
+	// Analysis Application samples based on TackleTestAppPublic
+	TackleTestApp = api.Application{
+		Name: "Tackle Testapp public",
+		Repository: &api.Repository{
+			Kind: "git",
+			URL:  "https://github.com/konveyor/tackle-testapp-public",
+			Branch: "main",
+		},
+	}
+	TackleTestAppGradle = api.Application{
+		Name: "Tackle Testapp public gradle",
+		Repository: &api.Repository{
+			Kind: "git",
+			URL:  "https://github.com/konveyor/tackle-testapp-public",
+			Branch: "gradle",
```

Comment:
I'm not sure if the is right to have gradle branch or some other solution, feedback is welcome.

I was just thinking that having different test applications for different releases is not very handy for tests debugging and stability. Feeling as better to be able say that TestAppXY analysis works on release-0.2 and 0.3, but is broken on 0.4, than having the "same" application that behave differently across releases.